blog/views.py

```diff
@@ -1,3 +1,53 @@
 """Views for the blog app."""
+from django.shortcuts import get_object_or_404, redirect, render
+from django.utils import timezone
 
-# Create your views here.
+from .forms import PostForm
+from .models import Post
+
+
+def post_list(request):
```

Comment:
This is unrelated to the PR purpose. You need to merge #25/#24 first.
To simplify things, move pylint integration commit into a separate PR and let's merge it before other things.